content/en/real_user_monitoring/mobile_and_tv_monitoring/web_view_tracking/_index.md

```diff
@@ -194,7 +219,7 @@ Note that `JavaScriptMode.unrestricted` is required for tracking to work on Andr
 
 ### Access your web views
 
-Your web views appear in the [RUM Explorer][5] with associated `service` and `source` attributes. The `service` attribute indicates the web component the web view is generated from, and the `source` attribute denotes the mobile application's platform, such as Android.
+Your web views appear in the [RUM Explorer][1] with associated `service` and `source` attributes. The `service` attribute indicates the web component the web view is generated from, and the `source` attribute denotes the mobile application's platform, such as Android.
 
 Filter on your Android and Android TV applications, and click a session. A side panel with a list of events in the session appears.
```

Comment:
Filter how?

Reply:
Thanks for this callout. Added instructions.